StringCalculatorProject/Program.cs

```diff
@@ -8,11 +8,26 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            Console.WriteLine("Enter string contains any amout of numbers: ");
-            string numbers = "2,3\n4";
+            Console.WriteLine("Enter string with delimiters: [yes, no]");
+            bool WithDelimeter = Console.ReadLine() == "yes"? true: false;
+            
+            if (WithDelimeter)
+            {
+                Console.WriteLine("Enter string:");
+                string delimiter = Console.ReadLine();
+                Console.WriteLine();
+                string numbers = Console.ReadLine();
+                var sc = new StringCalculator();
+                Console.WriteLine(sc.Add(numbers, new char[] { delimiter[2] }));
```

Comment:
both delimiters and numbers should be in the same string, not 2 different inputs